soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -293,13 +313,6 @@ static ALWAYS_INLINE void clock_init(void)
 	GPC_CM_SetNextCpuMode(1, kGPC_RunMode);
 	GPC_CM_EnableCpuSleepHold(0, false);
 	GPC_CM_EnableCpuSleepHold(1, false);
-
-#if !defined(CONFIG_PM)
-	/* Enable the AHB clock while the CM7 is sleeping to allow debug access
-	 * to TCM
-	 */
-	BLK_CTRL_S_AONMIX->M7_CFG |= BLK_CTRL_S_AONMIX_M7_CFG_TCM_SIZE_MASK;
```

Comment:
This code shouldn't have been added in the first place. We can't modify the default value recorded in M7_CFG, otherwise, edma driver can't read correct data from this register and do correct address transition.